source/mongofiles/mongofiles-examples.txt

```diff
@@ -0,0 +1,209 @@
+.. _mongofiles-examples:
+
+=====================
+mongofiles Examples
+=====================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. facet::
+   :name: programming_language
+   :values: shell
+
+.. |arrow| unicode:: U+27A4
+.. |tool-binary| replace:: ``mongofiles``
+
+This page shows examples for |tool-binary|.
+
+Run |tool-binary| from the system command line, not the 
+:binary:`~bin.mongo` shell.
```

Comment:
As part of these updates can we edit this line to refer to the MongoDB shell, since the mongo shell is deprecated?